Bio_Seq_Analysis_Tool.py

```diff
@@ -0,0 +1,163 @@
+from typing import Union, Tuple, List, Set, Dict
+from modules_for_BSAT import fastq_analysis as fq
+from modules_for_BSAT  import dna_rna_analysis as na
+from modules_for_BSAT import protein_analysis as pa
+
+
+def dna_rna_analysis(*args: str, operation: str) -> Union[List[float], List[str]]:
+    """
+    This function performs a number of operations on DNA or RNA.
+    
+    Operations supported by this functions:
+    -transcribe - return transcribed sequence
+    -reverse - return reverse sequence
+    -complement - return complement sequence
+    -reverse_complement - return reverse complement sequence
+    -gc_calculate - return sequence GC-content in percent
+    
+    :param args: nucleic acid sequence
+    :type param seqs: str
+    
+    :param operation: type of operation required
+    :type param operation: str
+    
+    :return: Analysis of nucleic acid sequence
+    :rtype: List[str]
+
+    :raises ValueError: if sequence not RNA or DNA, also if the operation value out of OPERATION_DICT
+    """
+    OPERATION_DICT = {'transcribe': na.transcribe, 
+                'reverse': na.reverse,
+                'reverse_complement': na.reverse_complement,
+                'complement': na.complement,
+                'gc_calculate': na.gc_calculate}
+    analysis = []
+    for seq in args:
+        na.is_na(seq)
+        if operation in OPERATION_DICT.keys():
+            analysis.append(OPERATION_DICT[operation](seq))
+        else:
+            raise ValueError(f'Wrong operation!')
```

Comment:
```suggestion
            raise ValueError(f'Unsupported operation {operation}!')
```
Чуть добавим заботы о пользователе:)